badge/badge.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Badge</title>
+    <link href="badge.css" rel="stylesheet" type="text/css" />
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+        <link rel = "stylesheet" href = /avatar/avatar.css>
+  </head>
+  <body>   
+    <div class = "icon-container">
+        <div class = "badge-on-icon">
+            <img src = "/src\cart.svg" class = "badge-icon" alt="cart">
+            <span class = "badge badge-red">5</span>
+        </div>
+        <div class = "badge-on-icon">
+            <img src = "/src\msg.svg" class = "badge-icon"  alt="cart">
+            <span class = "badge badge-green">5</span>
+        </div>
+        <div class = "badge-on-icon">
+            <img src = "/src\cart.svg" class = "badge-icon"  alt="cart">
+            <span class = "badge badge-black">5</span>
+        </div>
+        <div class = "badge-on-icon">
+            <p  class = "badge-icon text"> text
+            <span class = "badge badge-purple">5</span>
+        </p>
+        </div>
+    </div>
+    <div class = "avatar">
+        <div class = "avatar-with-status">
+            <img class = "avatar-circular avatar-s" src="/src\avatar.jpg" alt="avatar image">
+            <span class = "status status-green"></span>
```

Comment:
in alt tag.. just write avatar..instead of avatar image..because it is evident that its a image

Reply:
sure - updated it